internal/impl/kube.go

```diff
@@ -37,16 +39,21 @@ import (
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
+	"sigs.k8s.io/yaml"
 )
 
 const (
 	// Name of the container that hosts the application binary.
 	appContainerName = "serviceweaver"
+
 	// The exported port by the Service Weaver services.
 	servicePort = 80
 
 	// Port used by the weavelets to listen for internal traffic.
 	internalPort = 10000
+
+	// Default Prometheus port.
+	prometheusPort = 9090
```

Comment:
is this also going away? If I remember correctly, we discussed to export only logs (to stdout) by default.

Reply:
Yeah, I'll remove this in a future PR!